content/docs/0.8.x/integrations/migrate_from_070_to_080/index.md

```diff
@@ -0,0 +1,739 @@
+```golang
+import (
+    "github.com/cloudevents/sdk-go/pkg/cloudevents"
+)
+```
+should be replaced with
+```golang
+import (
+ 	cloudevents "github.com/cloudevents/sdk-go/v2" // make sure to use v2 cloudevents here
+)
+```
+
+### All Keptn CloudEvents have appropriate constants and structs
+
+Considering you have imported the updated go-utils and CloudEvents like this
+
+```golang
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+    cloudevents "github.com/cloudevents/sdk-go/v2" // make sure to use v2 cloudevents here
+    keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
+)
+
+// ServiceName specifies the current services name (e.g., used as source when sending CloudEvents)
+const ServiceName = "keptn-service-template-go"
+```
+
+a series of constants, functions, and structs will be available for processing Keptn CloudEvents, which is demonstrated by the following example:
+
+```golang
+func parseKeptnCloudEventPayload(event cloudevents.Event, data interface{}) error {
+	err := event.DataAs(data)
+	if err != nil {
+		log.Fatalf("Got Data Error: %s", err.Error())
+		return err
+	}
+	return nil
+}
+
+func processKeptnCloudEvent(ctx context.Context, event cloudevents.Event) error {
+    switch event.Type() {
+	// sh.keptn.event.get-sli
+	case keptnv2.GetTriggeredEventType(keptnv2.GetSLITaskName): // sh.keptn.event.get-sli.triggered
+		log.Printf("Processing Get-SLI.Triggered Event")
+
+		eventData := &keptnv2.GetSLITriggeredEventData{} // Data Structure for Get-SLI Triggered Event
+		parseKeptnCloudEventPayload(event, eventData)
+
+		return HandleGetSliTriggeredEvent(myKeptn, event, eventData)
+
+	// sh.keptn.event.deployment
+	case keptnv2.GetTriggeredEventType(keptnv2.DeploymentTaskName): // sh.keptn.event.deployment.triggered
+		log.Printf("Processing Deployment.Triggered Event")
+
+		eventData := &keptnv2.DeploymentTriggeredEventData{}
+		parseKeptnCloudEventPayload(event, eventData)
+
+		return HandleDeploymentTriggeredEvent(myKeptn, event, eventData)
+        }
+    return nil
+}
+
+func HandleGetSliTriggeredEvent(myKeptn *keptnv2.Keptn, incomingEvent cloudevents.Event, data *keptnv2.GetSLITriggeredEventData) error {
+	log.Printf("Handling get-sli.triggered Event: %s", incomingEvent.Context.GetID())
+	return nil
+}
+
+func HandleDeploymentTriggeredEvent(myKeptn *keptnv2.Keptn, incomingEvent cloudevents.Event, data *keptnv2.DeploymentTriggeredEventData) error {
+	log.Printf("Handling deployment.triggered Event: %s", incomingEvent.Context.GetID())
+	return nil
+}
+```
+
+*Note*: The full source code of those example is available in our [keptn-service-template-go](https://github.com/keptn-sandbox/keptn-service-template-go).
+
+### Sending CloudEvents
+
+While in keptn/go-utils 0.7.x you had to take care of sending CloudEvents in the correct format, we have tried to hide this complexity by introducing helper functions in keptn/go-utils 0.8.x.
```

Comment:
please change it to: > While in keptn/go-utils 0.7.x you had to take care of sending CloudEvents in the correct format, helper functions in keptn/go-utils 0.8.x hide this complexity now.